.circleci/config.yml

```diff
@@ -1,10 +1,52 @@
 version: 2
 jobs:
+  litmus_daily:
+    machine: true
+    steps:
+      - attach_workspace:
+          at: ~/project
+      - run: docker login -u=$QUAY_USER -p=$QUAY_PASS quay.io
+      - run: docker run --entrypoint "./run_litmus_tests_oss.sh" -e ONE_TEST=src/cloud/rest_api/smoke/test_smoke.py -e PROTOSPATH=/Litmus/result/influxd_test -e BINARYPATH=/Litmus/result/bin/linux/influxd -e BOLTPATH=/Litmus/result/influxd_test/influxd.bolt -e ENGINEPATH=/Litmus/result/influxd_test --net host -v /var/run/docker.sock:/var/run/docker.sock -v ~/project:/Litmus/result quay.io/influxdb/litmus:latest
+      - run:
+          name: Litmus Smokte Tests Success
+          when: on_success
+          command: |
+            curl -X POST https://slack.com/api/chat.postMessage \
```

Comment:
Would you turn these slack posts into a script in etc?

Reply:
Would this orb work?

https://circleci.com/orbs/registry/orb/circleci/slack

Reply:
@gshif Yeah, I like @BondAnthony 's orb idea for sure!

Reply:
Since slack notifications for litmus nightly job are already there, I would not want to make the changes right now. I can create the ticket to look into that when I have more time. My priority is to add more tests. Would that work?

Reply:
Nah... I think you should do them now.  The reason is that it'll set the pattern for other builds sending to slack.  This will lead us in a really positive direction.

Reply:
Why don't we use the slack notifications that are already there? This PR is about adding Litmus daily test runs and I do not want to hold it because of notifications.

Reply:
Created two shell scripts to be used for slack posts.

Reply:
Re: orbs suggested by @BondAnthony. It is not just import and use. It requires changes to CircleCI configuration, that I would like to leave to SREs.